src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -2405,7 +2423,7 @@ static sd_dhcp_client *dhcp_client_free(sd_dhcp_client *client) {
 
 DEFINE_TRIVIAL_REF_UNREF_FUNC(sd_dhcp_client, sd_dhcp_client, dhcp_client_free);
 
-int sd_dhcp_client_new(sd_dhcp_client **ret, int anonymize) {
+int sd_dhcp_client_new(sd_dhcp_client **ret, int anonymize, int bootp) {
```

Comment:
Please do not add a new argument. Instead, please introduce a new function to control the flag.

Reply:
That makes sense. I'll change it.